src/pages/Dashboard/Tweet/index.tsx

```
@@ -0,0 +1,67 @@
+import { zodResolver } from '@hookform/resolvers/zod'
+import * as Sentry from '@sentry/react'
+import type React from 'react'
+import { useForm } from 'react-hook-form'
+import { z } from 'zod'
+
+import { Error } from '@/src/components/Error'
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)